### PR TITLE
Clarify interpreter webhook service discovery usage

### LINKED ISSUE #5958: Cannot access karmada-interpreter-webhook by configuring Service in ResourceInterpreterWebhookConfiguration
**What happened**:

It was found that since v1.6, users cannot access `karmada-interpreter-webhook` component by configuring Service in ResourceInterpreterWebhookConfiguration according to the guidance in https://github.com/karmada-io/karmada/tree/master/examples/customresourceinterpreter, such as configuration:

```yaml
apiVersion: config.karmada.io/v1alpha1
kind: ResourceInterpreterWebhookConfiguration
metadata:
  name: examples
webhooks:
- clientConfig:
    caBundle: {{CABundle}}
    service:
      namespace: karmada-system
      name: karmada-interpreter-webhook-example
      port: 443
      path: /interpreter-workload
  interpreterContextVersions:
  - v1alpha1
  name: workloads.example.com
  rules:
  - apiGroups:
    - workload.example.io
    apiVersions:
    - v1alpha1
    kinds:
    - Workload
    operations:
    - InterpretReplica
    - ReviseReplica
    - Retain
    - AggregateStatus
    - InterpretHealth
    - InterpretStatus
    - InterpretDependency
  timeoutSeconds: 3
```

**What you expected to happen**:

Can access karmada-interpreter-webhook by configuring Service in ResourceInterpreterWebhookConfiguration.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

This issue was introduced by #2999

**Environment**:
- Karmada version: After v1.6
- kubectl-karmada or karmadactl version (the result of `kubectl-karmada version` or `karmadactl version`):
- Others:

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

In order to guide users to better use the resource interpreter webhook, the configuration description of clientConfig is added, and the directory description is also added.

**Which issue(s) this PR fixes**:
Fixes #5958

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: examples/customresourceinterpreter/README.md [hunk 1]
```diff
@@ -86,9 +113,63 @@ spec:
...
+      caBundle: {{caBundle}}
+    interpreterContextVersions: [ "v1alpha1" ]
+    timeoutSeconds: 3
+```
+
+If you only need to access the resource interpreter webhook service in the `Karmada-host` cluster, you can directly configure `clientConfig` with the Service domain name in the cluster:
+
+```yaml
+clientConfig:
+  url: https://karmada-interpreter-webhook-example.karmada-system.svce:443/interpreter-workload
```
Comment (lxtywypc) [inline examples/customresourceinterpreter/README.md:143]: I'm not sure but is `.svce:443` right?

**Review by lxtywypc — COMMENTED** [review]
And another point I'm not sure, if we could deploy the load balancer through karmada-apiserver?

File: examples/customresourceinterpreter/README.md (hunk 1 above)
Comment (XiShanYongYe-Chang) [reply examples/customresourceinterpreter/README.md:143]: Nice finding, a typo here, it should be `.svc:443`

**Comment by XiShanYongYe-Chang** [comment]
> And another point I'm not sure, if we could deploy the load balancer through karmada-apiserver?

This test has not been performed. I understand that the cluster IP address in the service of karmada-apiserver should be fake. NodePort or LoadBalancer depend on this cluster IP address and cannot be accessed.

**Review by RainbowMango — COMMENTED** [review]
/assign

**Comment by RainbowMango** [comment]
/retitle Clarify interpreter webhook service discovery usage

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**